src/app/shared/components/breadcrumbs/breadcrumbs/breadcrumbs.module.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { BreadcrumbsComponent } from './breadcrumbs.component';
+
+
+
+@NgModule({
+  declarations: [
+    BreadcrumbsComponent
+  ],
+  imports: [
+    CommonModule
+  ]
```

Comment:
add BreadcrumbsComponent to exports: [] as well to use this component outside of current module